cypress/support/commands.js

```diff
@@ -1,3 +1,9 @@
+Cypress.Commands.add('onLocal', tests => {
```

Comment:
Nice but introducing multiple ways of doing this, please move this into another PR and we'll get that and the equivalent for onLocalAndTest) merged separately from this PR. That PR should then remove the two describe by env helpers.

Reply:
Chatted on slack, decided not to use this in this PR #ViveLaRevolution